go.mod

```diff
@@ -37,9 +37,9 @@ require (
 	golang.org/x/text v0.14.0
 	gopkg.in/yaml.v2 v2.4.0
 	gotest.tools/v3 v3.5.1
-	k8s.io/api v0.29.3
-	k8s.io/apimachinery v0.29.3
-	k8s.io/client-go v1.5.2
+	k8s.io/api v0.29.2
+	k8s.io/apimachinery v0.29.2
+	k8s.io/client-go v0.29.2
```

Comment:
It might be a good idea to keep patch version to `v0.29.3`, not sure if is even more recent one released.

Reply:
fixed